README.md

```diff
@@ -21,3 +21,21 @@ zipArtifact := pillar lastSuccessfulBuild runs anyOne artifacts detect: [ :artif
 
 ZnClient new url: zipArtifact url; downloadTo: 'Pillar.zip'
 ```
+
+### Use a Jenkins protected by a password:
+
+```smalltalk
+| server job zipArtifact | 
+
+server := JenkinsServer name: 'aServerName' url: ('https://yourURL' asUrl) username: 'x.y@something.com'; token: 'aToken'; yourself.
+
+"Your username should be the email address of your Jenkins account.
+You can find your token in the settings of your account.
+Account -> Configure -> API Token"
+
+job := server jobs detect: [ :job | job name = 'aJobName' ].
```

Comment:
I just take the same code as as before (to get the Pillar.zip) to give an example of how to get a zip file on a secure server.